docs/quick-start/development.md

```diff
@@ -29,6 +29,7 @@ To check that the build is working, run `docker-compose ps`. This should give yo
 vaas_backend-statuses_1     /wait-for-it.sh uwsgi:3030 ...   Up
```

Comment:
If we currently gather statuses via celery task I suppose that backend_statuses container is no longer needed.